src/content/en/updates/2020/03/devtools.md

```diff
@@ -0,0 +1,153 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Emulate color vision deficiencies, Dock To Left in the Command Menu, and more.
+
+{# wf_updated_on: 2020-03-10 #}
+{# wf_published_on: 2020-03-10 #}
+{# wf_tags: chrome82, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Emulate color vision deficiencies, Dock To Left in the Command Menu, and more. #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 82) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Emulate vision deficiencies {: #vision-deficiencies }
+
+Open the [Rendering tab](/web/tools/chrome-devtools/evaluate-performance/reference#rendering)
+and use the new **Emulate vision deficiencies** feature to get a better idea of how people with
+different types of vision deficiencies experience your site.
+
+<figure>
+  <img src="/web/updates/images/2020/03/vision.png"
+       alt="Emulating blurred vision."/>
+  <figcaption>
+    Emulating blurred vision.
+  </figcaption>
+</figure>
+
+DevTools can emulate blurred vision and the following
+[types of color vision deficiencies](http://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/):
+
+* Protanopia. The inability to perceive red light.
+* Protanomaly. A reduced sensitivity to red light.
+* Deuteranopia. The inability to perceive green light.
+* Deuteranomaly. A reduced sensitivity to green light.
+* Tritanopia. The inability to perceive blue light.
+* Tritanomaly. A reduced sensitivity to blue light (extremely rare).
+* Achromatopsia. The inability to perceive any color except for shades of grey (extremely rare).
+* Achromatomaly. A reduced sensitivity to green, red, and blue light (extremely rare).
```

Comment:
We should probably point out that the `-omaly` forms are the (rare) extreme versions of the forms ending in `-ia`, and that every person with an `-ia` vision deficiency is different and might see things differently (being able to perceive a bit more/less of the relevant colors). These simulations are just to give you a visual approximation of how someone might experience this — and although the approximation should be good enough for you to identify and resolve issues, **there’s no way to simulate exactly what a given person would experience**.

Reply:
(This is based on Twitter feedback)

Reply:
Done